templates/webpack.config.cjs

```diff
@@ -35,6 +35,18 @@ module.exports = (env) => {
     throw new Error('Missing platform');
   }
 
+  /**
+   * Using Module Federation might require disabling hmr.
+   * Uncomment below to set `devServer.hmr` to `false`.
+   *
+   * Keep in mind that `devServer` object is not available
+   * when running `webpack-bundle` command and crash
+   * because of reading from `undefined` will occur. 
```

Comment:
```suggestion
   * Keep in mind that `devServer` object is not available
   * when running `webpack-bundle` command. Be sure
   * to check its value to avoid accessing undefined value,
   * otherwise an error might occur.
```

I think that is clearer to understand.